processing/src/main/java/org/apache/carbondata/processing/newflow/DataLoadProcessBuilder.java

```diff
@@ -52,10 +53,15 @@
 
   public AbstractDataLoadProcessorStep build(CarbonLoadModel loadModel, String storeLocation,
       CarbonIterator[] inputIterators) throws Exception {
+    boolean batchSort = Boolean.parseBoolean(CarbonProperties.getInstance()
+        .getProperty(CarbonCommonConstants.LOAD_USE_BATCH_SORT,
```

Comment:
Yes, is it possible to add it in load option in this PR? If not, in create table is fine.